packages/cli/src/lib/defaults/build-strategies/wasm/rust/vm/Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM rust:1.65.0 as base
+FROM rust:1.66-alpine as base
 
 # Install the wasm32 rust build target
 RUN rustup target add wasm32-unknown-unknown
 
 WORKDIR /build-deps
 
 # Install curl
-RUN apt-get update
-RUN apt-get -y install curl clang llvm build-essential
+RUN apk add curl build-base pkgconfig openssl-dev bash
```

Comment:
Hey @Niraj-Kamdar, apologies for not asking about this before, but why are we now installing `build-base`, `pkgconfig`, `openssl-dev` and `base`?

Also I should have caught this before, but for rust builds we require `clang`, `llvm` and `build-essentials` to build & link native C libs.

Reply:
One answer I found is that `build-base` is the alpine equivalent of `build-essentials`.